### ISSUE TITLE
Channel creation timeout in neutrino mode without feeurl

### ISSUE THREAD

**Issue by Sjors** [lightningnetwork/lnd#4770]
I tried opening a channel from Lnd to c-lightning, as well as the other way around. It would just timeout after 5 minutes with no explanation.

I configured the node to use Neutrino and connected it to my Bitcoin Core node only.

Once I configured `feeurl` and restarted Lnd the channel creation, which I initiated earlier, went ahead.

It should probably complain in the `openchannel` command or in the logs (when the other side opens) that it needs fee information.

**Comment by Sjors** [comment]
As an aside, does Lnd broadcast its (channel creation) transactions to the neutrino peer? So far it didn't seem to and I had to manually broadcast the raw transaction.

**Comment by Roasbeef** [comment]
> As an aside, does Lnd broadcast its (channel creation) transactions to the neutrino peer

If it's the initiator (which is the only person that has the funding transaction), then yeah it _should_. Did you see it _attempt_ to do so in the logs? It'll also rebroadcast any unconfirmed transactions on start up as well.

**Comment by Sjors** [comment]
I initiated several channels from the Lnd node, none of the funding transactions were broadcast.

The log shows:
```
2020-11-14 17:48:08.267 [INF] FNDG: Initiating fundingRequest ...
2020-11-14 17:48:08.267 [INF] CHFD: Performing funding tx coin selection using 2500 sat/kw as fee rate
2020-11-14 17:48:08.287 [INF] FNDG: Target commit tx sat/kw for pendingID(...
2020-11-14 17:48:08.287 [INF] FNDG: Starting funding workflow with ...:9735 for pending_id(...), committype=tweakless
2020-11-14 17:48:08.421 [INF] FNDG: Recv'd fundingResponse for pending_id(...)
2020-11-14 17:48:08.429 [INF] FNDG: pendingChan(...): remote party proposes num_confs=3, csv_delay=144
2020-11-14 17:48:08.431 [INF] FNDG: Generated ChannelPoint(...:0) for pending_id(...)
2020-11-14 17:48:08.550 [INF] FNDG: Broadcasting funding tx for ChannelPoint(...:0): ...
2020-11-14 17:48:08.550 [INF] LNWL: Inserting unconfirmed transaction ...
````

Maybe I wasn't patient enough, but it didn't show up in the mempool of the only peer it's connected to (my own bitcoind node) and I had to use sendrawtransaction.

**Comment by Roasbeef** [comment]
Which version are you on?

**Comment by Roasbeef** [comment]
Also which bitcoind version were you running?

**Comment by Sjors** [comment]
Lnd `0.11.1-beta commit=v0.11.1-beta` against bitcoind v0.20.99.0-1d3ec2a1f (master branch, slightly before the new address  message was merged, see https://github.com/btcsuite/btcd/issues/1661

**Comment by bitcoinlizard** [comment]
I have observed the same behavior as @Sjors.  I am using the current LND from the master branch: 
"0.11.99-beta commit=v0.11.0-beta-534-g7e298f143425337448f95019234b22849d64aca3-dirty" 

I am running btcd version 0.21.0-beta acting as the neutrino server.  I don't see anything in the btcd log to indicate that LND attempted to broadcast the transaction.  I needed to retrieve the raw_tx_hex with "lncli listchaintxns" and broadcast the transaction by other means.  This has occurred for channel creation transactions as well as regular on chain transactions and in very limited testing seems to occur 100% of the time.  

Most likely unrelated but I am connecting to btcd via .onion address.
